### PR TITLE
user data clean up

### LINKED ISSUE #5698: user data clean up
once we remove the temp fastly headers we will need to update the language and country for users with these countries
- [ ] xx
- [ ] xr

### PR DESCRIPTION
**Pull request by katiecrane**
fixes #5698 

db updates to change brazil and mexico users to the proper language and country code

### CODE REVIEW

File: lib/modules/dosomething/dosomething_global/dosomething_global.install [hunk 1]
```diff
@@ -28,3 +28,45 @@
...
+  */
+function dosomething_global_update_7003(&$sandbox) {
+  $results = db_query("SELECT u.uid, u.mail, a.field_address_country, u.language
+                        FROM users u
+                        INNER JOIN field_data_field_address a on a.entity_id = u.uid
+                        WHERE u.language = 'en-global'
+                        AND a.field_address_country = 'US';");
+
+  foreach ($results as $result)
+  {
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_global/dosomething_global.install]: in drupal brackets should be on the line above `foreach (blah as blah) {`

File: lib/modules/dosomething/dosomething_global/dosomething_global.install [hunk 2]
```diff
@@ -28,3 +28,45 @@
...
+    $user = user_load($result->uid);
+    $user->language = 'english';
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_global/dosomething_global.install]: does this work? shouldn't it be the code `en`?

File: lib/modules/dosomething/dosomething_global/dosomething_global.install [hunk 3]
```diff
@@ -28,3 +28,45 @@
...
+/**
+ * change BR users' language from global english to pt-br, country from XR to BR 
+ */
+function dosomething_global_update_7005(&$sandbox) {
+  $results = db_query("UPDATE users as u
+                        INNER JOIN field_data_field_address a on a.entity_id = u.uid
+                        SET u.language = 'pt-br', a.field_address_country = 'BR', a.language = 'pt-br'
+                        WHERE u.language = 'en-global'
+                        AND a.field_address_country = 'XR';");
+
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_global/dosomething_global.install]: you can do these all in the same update hook, they don't need to be 3 different ones

File: lib/modules/dosomething/dosomething_global/dosomething_global.install (hunk 2 above)
Comment (katiecrane) [inline lib/modules/dosomething/dosomething_global/dosomething_global.install]: it doesn't break but yeah the code should be `en`

**Comment by angaither** [comment]
:+1:

**Pull request merged**